xcube_cci/cciodp.py

```diff
@@ -1107,6 +1113,9 @@ async def _get_variable_infos_from_feature(self, feature: dict, session) -> (dic
             if '_FillValue' in variable_infos[fixed_key]:
                 variable_infos[fixed_key]['fill_value'] = variable_infos[fixed_key]['_FillValue']
                 variable_infos[fixed_key].pop('_FillValue')
+            else:
+                variable_infos[fixed_key]['fill_value'] = \
+                    self._determine_fill_value(dataset[key].dtype)
```

Comment:
You can only assume `NaN` for floating point types. For integer types, you cannot make a guess because this is wrong as my example above shows.